src/setup/devnet/templates/docker/docker-compose.yml.njk

```diff
@@ -75,7 +75,7 @@ services:
     environment:
       - HEIMDALL_URL=http://heimdall{{ node }}:1317
     volumes:
-      - ./devnet/node{{ node }}/bor:/root/.bor
```

Comment:
Not sure but shouldn't the docker volume be `/root/var/lib/bor` ? For heimdall it's `/root/var/lib/heimdall`.

Reply:
You are right. Done, [here](https://github.com/maticnetwork/matic-cli/pull/215/commits/80a4181e29019ffe3b0d5454b56f13689cff4f33), thanks.